roles/ceph-common/files/cephstable.asc

```diff
@@ -0,0 +1,31 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
This is used to fetch the ceph version when it runs `sudo apt-get install ceph`.